rules/framework.bzl

```diff
@@ -284,10 +300,25 @@ def _apple_framework_packaging_impl(ctx):
         cc_info_provider,
         swift_common.create_swift_info(**swift_info_fields),
         DefaultInfo(files = depset(framework_files)),
+        AppleBundleInfo(
+            archive = None,
+            archive_root = None,
+            binary = binary_out,
+            bundle_id = bundle_id,
+            bundle_name = framework_name,
+            bundle_extension = bundle_extension,
+            entitlements = None,
+            infoplist = infoplist,
+            minimum_os_version = str(current_apple_platform.target_os_version),
+            platform_type = str(current_apple_platform.platform.platform_type),
+            product_type = ctx.attr._product_type,
+            uses_swift = swiftmodule_out != None,
```

Comment:
nit, can you add parens here to keep the order of operations obvious?
suggestion:
uses_swift = (swiftmodule_out != None),

Reply:
I hope the RHS doesn't turn into an array/tuple if you do that!